front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/AngularServlet.java

```diff
@@ -1,42 +1,42 @@
-/*
- * Copyright 2017 JBoss Inc
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.apicurio.studio.fe.servlet.servlets;
-
-import java.io.IOException;
-
-import javax.servlet.GenericServlet;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
-
-/**
- * A simple servlet that forwards everything to /index.html.
- *
- * @author eric.wittmann@redhat.com
- */
-public class AngularServlet extends GenericServlet {
-
-    private static final long serialVersionUID = 4259630009438256847L;
-
-    /**
-     * @see javax.servlet.GenericServlet#service(javax.servlet.ServletRequest, javax.servlet.ServletResponse)
-     */
-    @Override
-    public void service(ServletRequest req, ServletResponse res) throws ServletException, IOException {
-        req.getRequestDispatcher("/index.html").forward(req, res); //$NON-NLS-1$
-    }
-
+/*
+ * Copyright 2017 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.studio.fe.servlet.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.GenericServlet;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+/**
+ * A simple servlet that forwards everything to /index.html.
+ *
+ * @author eric.wittmann@redhat.com
+ */
+public class AngularServlet extends GenericServlet {
+
+    private static final long serialVersionUID = 4259630009438256847L;
+
+    /**
+     * @see javax.servlet.GenericServlet#service(javax.servlet.ServletRequest, javax.servlet.ServletResponse)
+     */
+    @Override
+    public void service(ServletRequest req, ServletResponse res) throws ServletException, IOException {
+        req.getRequestDispatcher("/index.html").forward(req, res); //$NON-NLS-1$
+    }
+
 }
```

Comment:
Was something changed here?  I'm not sure why the diff is messed up...

Reply:
Nothing changed